test/test_typed_array.cpp

```diff
@@ -360,18 +360,18 @@ TEST_SUITE("typed_array")
 
     TEST_CASE_TEMPLATE_INVOKE(
         all,
-        bool,
+        /*bool,
```

Comment:
I suppose that was temporary? The current build hasnt been tested yet will all the types below then.